src/config/config.ts

```diff
@@ -258,14 +233,15 @@ function compressRequestBodies(
         resolve: any,
         errorHandlers: any[]
     ) => {
-        if (
-            method === 'POST' &&
-            urlsToCompress.filter(pair =>
-                pairMatchesPath(pair, url, queryParameters)
-            ).length > 0
-        ) {
-            headers['Content-Encoding'] = 'gzip';
-            body = pako.gzip(JSON.stringify(body)).buffer;
+        if (method === 'POST') {
+            var bodyString = JSON.stringify(body);
+            if (bodyString.length > REQ_BODY_SIZE_CHAR_LIMIT) {
```

Comment:
How expensive would it be to do `new Blob([bodyString]).size`? Using char count to estimate size is a bit unreliable.

Reply:
@Luke-Sikina it's just a heuristic right?  and why is it innacurate?

Reply:
@alisman I also reasoned that a precise estimate is not needed here. We just need the least resource-intensive estimate of request size.